### ISSUE TITLE
NestedScrollview+ SliverToBoxAdapter +  Webview + TabBarView + BackdropFilter + ImageFilter.blur rendering issue on iOS

### ISSUE THREAD

**Issue by pawarn** [flutter/flutter#67713]
I am getting the issue to render BackdropFilter with web view inside the SliverToBoxAdapter for iOS it's blurring all the widget till the SliverToBoxAdapter web view not disappear.

     NestedScrollView(

          headerSliverBuilder: (context, innerBoxIsScrolled){
            return [
            SliverToBoxAdapter(
              child: SizedBox(
                height: 60,
              ),
            ),
            SliverToBoxAdapter(
              child: Webview(),  // with some content on the top of the tabs
            ),
              SliverToBoxAdapter(
                child: _tabBar(), // this has no of tabs
              )
            ];
          },
          body: TabBarView(
            children: selectedTabs, // list of tabs content to be diasplay
            controller: _tabController,
          ),
        )

Inside the tabs has BackdropFilter with blur image. it's working if we scroll above after web view hide below the toolbar

Flutter Doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.0, on Mac OS X 10.15.5 19F101, locale en-IN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
[✓] Android Studio (version 4.0)
[✓] Connected device (1 available)

**Comment by darshankawar** [comment]
@pawarn 
Please provide complete reproducible code sample that we can simply copy paste and run in IDE that shows the issue.
Does the same code work in Android ? Can you also provide the video of both platforms ?
Thanks.

**Comment by pawarn** [comment]
@darshankawar 
Okay, I'll try to provide you complete working code because it has an API integration and the same will not work for you, and Yes same code is working on android.
[Uploaded video on the link](https://drive.google.com/file/d/1a5KLjtqVRVJFOfqEUzadV43F9fql9bA1/view?usp=sharing)

**Comment by pawarn** [comment]
Hi @darshankawar 
did you found any solution for this?

**Comment by darshankawar** [comment]
@pawarn 
Just sharing the video will only help to certain extent, but most importantly we want a complete reproducible code sample that can help to identify the issue, as requested earlier.
Thanks,

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by GaspardMerten** [comment]
I had the same issue with my app and I found the solution?

Turns out that BackDropFilter as its name suggests makes anything under it blurred. And when you combine that with the way the TabBarView arrange its children in the widget's tree, you get that undesirable behavior...

My solution is to use ImageFiltered and Stack together.

`Stack(children: [ImageFiltered(child: a background image for instance), MyWidgetWithoutBlur()])`
